src/index.js

```diff
@@ -24,7 +25,7 @@ let clientFactory = require('./client-factory')
  *
  * @returns {Promise<function>} Client async function
  */
-module.exports = async function awsLite (config = {}) {
+module.exports = async function awsLite (config = { verifyService: true }) {
```

Comment:
it could even be `config.verifyService ??= true` but I don't want to mess with ESLint in this scope. Here's a related PR though 😅 https://github.com/architect/eslint-config/pull/6

Reply:
I ended up moving this default to the client function where validation happens. Mostly I did this to satisfy nyc code coverage as it was marking the `validateService` function on line 212 as uncovered.
I know the function is executing and that it throws both errors in various scenarios because I have tests for both.